### ISSUE TITLE
Cvd-contract-type - wrong hungarian translation for 'veh-acq'

### ISSUE THREAD

**Issue by tompeeszter** [OP-TED/eForms-SDK#1031]
We have noticed that in **cvd-contract-type** codlist in case of code _'veh-acq'_ the hungarian translation is not correct.

The english translation is:
```
         <Value ColumnRef="eng_label">
            <SimpleValue>Vehicle purchase, lease or rent</SimpleValue>
         </Value>
```

The hungarian one:
```
         <Value ColumnRef="hun_label">
            <SimpleValue>Járművásárlás, -bérlés vagy -bérlés</SimpleValue>
         </Value>
```

I think the proper hungarian translation should be like 'Járműbeszerzés, -lízing vagy -bérlés'

We use SDK 1.11.1, but it also a problem in SDK 1.12.
Could you please investigate this issue?

Thanks is advance!

**Comment by ferrakaop** [comment]
Thank you for raising this issue, there does seems to be a repeated term in Hungarian.  

The source of this codelist entry is in article 3 in the amendment to the CVD:
https://eur-lex.europa.eu/legal-content/HU/TXT/HTML/?uri=CELEX:32019L1161

The full sentence in Hungarian is "ajánlatkérő  szervek vagy közszolgáltató ajánlatkérők által odaítélt, közúti járművek  megvásárlására, lízingjére, bérletére vagy részletvételére vonatkozó  szerződések, amennyiben azok kötelesek alkalmazni"

Does your proposed correct align with the same words/terminology?

We can try to catch this correction SDK 1.13 and the label will later be patched to 1.11 and 1.12

**Comment by boglarkamakra** [comment]
Yes, our proposal complies with the legislation. Thank you very much for your support and for correcting the translation of the codelist!

**Comment by ferrakaop** [comment]
Thanks for confirming the codelist correction aligns with the directive.  We will include this in SDK 1.13 and you'll see it later on in the patches to the previous SDKs.
